doc/languages-frameworks/python.md

```diff
@@ -481,7 +481,7 @@ and the aliases
 #### `buildPythonPackage` function
 
 The `buildPythonPackage` function is implemented in
-`pkgs/development/python-modules/generic/default.nix`
+`pkgs/development/interpreters/python/buildpythonpackage.nix`
```

Comment:
As long as this nix file is named the same as the function it's defining (which I like :+1: ) could we give it the same capitalization as well?